### DISCUSSION TITLE
Feature Request: REST API for Issue Templates/Forms

### DISCUSSION THREAD

**Discussion by geromegrignon** [community/community#4423] [Projects and Issues]
Hello, the same way we have a Rest API for Issues, it would be awesome to get one for Issue Templates/Forms.

## Expectations

- to be able to list existing templates/forms from one repository
- to create a new template/forms
- to update existing template/forms

## Motivation

- to audit the repositories of an organization
- to retrieve the name of existing templates/forms (to validate the creation of new forms using my generator https://www.issue-forms-creator.app/new, the same way by adding a new forms the name is verified to avoid duplicates)
- to dispatch an existing template/forms on other repositories
- to add a new template/forms on multiple repositories

**Comment by Trimatix** [comment]
@github-staff, Has this feature request been triaged? This would be instrumental to a feature in one of my applications.

**Comment by pdelancie** [comment]
@github-staff It seems like a big disconnect to have forms and templates available for submitting issues when you're signed into GitHub, but to not be able to use forms or templates when submitting issues via the GitHub API. Seems like it wouldn't be that hard to make it so that if the body of the "Create an issue" call is structured in compliance with the YAML schema for a form then the resulting issue is created using the form layout and appears as a true form in the GitHub UI.
Is there any possibility of this happening? If so, when.
Please advise, thanks.

**Comment by pdelancie** [comment]
@github-staff This issue is almost 3 years old now. Is it not possible to give your customers the courtesy of a reply?
